common/lib/capa/capa/responsetypes.py

```diff
@@ -1957,7 +1977,30 @@ def validate_answer(self, answer):
             return False
 
     def get_answers(self):
-        return {self.answer_id: self.correct_answer}
+        _ = self.capa_system.i18n.ugettext
+        # Example: "Answer: Answer_1 or Answer_2 or Answer_3".
+        separator = u' <b>{or_separator}</b> '.format(
+            # Translators: Separator used in NumericalResponse to display multiple answers.
+            or_separator=_('or')
+        )
+        return {self.answer_id: separator.join([self.correct_answer] + self.additional_answers)}
+
+    def set_cmap_msg(self, student_answers, new_cmap, hint_type, hint_index):
+        """
+        Set cmap msg property.
```

Comment:
Can we use `Google Style` here for doc strings.  http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings